src/app/pages/settings-integration/billing/connected-account/settings-billing-account-dialog.component.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-useless-constructor */
+
+import { Component, Inject, OnInit } from '@angular/core';
+import { AbstractControl, FormControl, FormGroup } from '@angular/forms';
+import { MAT_DIALOG_DATA, MatDialog, MatDialogConfig, MatDialogRef } from '@angular/material/dialog';
+import { TranslateService } from '@ngx-translate/core';
+import { CentralServerService } from 'services/central-server.service';
+import { MessageService } from 'services/message.service';
+import { SpinnerService } from 'services/spinner.service';
+import { UsersDialogComponent } from 'shared/dialogs/users/users-dialog.component';
+import { BillingAccount } from 'types/Billing';
+import { ActionResponse } from 'types/DataResult';
+import { RestResponse } from 'types/GlobalType';
+import { Utils } from 'utils/Utils';
+
+import { DialogService } from '../../../../services/dialog.service';
+
+@Component({
```

Comment:
I think a connected-**accounts** folder for just the list would be extreme as it would contain only a single file. I am following the design which is similar to analytics
![image](https://user-images.githubusercontent.com/5779016/177265940-7060c792-8a64-4ad7-980d-fbd2edff4fa7.png)